os/arch/arm/src/imxrt/imxrt_pwm.c

```diff
@@ -1303,13 +1375,13 @@ FAR struct pwm_lowerhalf_s *imxrt_pwminitialize(int timer)
 			return NULL;
 		}
 
-		imxrt_pwm_drv_init3phpwm(timer);
+		imxrt_pwm_drv_init3phpwm(timer, DEFAULT_DUTYCYCLE, DEFAULT_FREQUENCY);
 
 		lower = (struct pwm_lowerhalf_s *)&g_pwm0_1;
 	} else
 #endif
 #ifdef CONFIG_IMXRT_PWM2
-	if (timer == 1) {
+		if (timer == 1) {
```

Comment:
please align code below.

Reply:
You should remove this adding a tab. Same as below.